src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets

```diff
@@ -2,7 +2,8 @@
 <Project>
   <PropertyGroup>
     <_NSwagCommand>$(NSwagExe)</_NSwagCommand>
-    <_NSwagCommand Condition="'$(MSBuildRuntimeType)' == 'Core'">$(NSwagExe_Core21)</_NSwagCommand>
+    <_NSwagCommand
+        Condition="'$(MSBuildRuntimeType)' == 'Core'">dotnet --roll-forward-on-no-candidate-fx 2 $(NSwagDir_Core21)/dotnet-nswag.dll</_NSwagCommand>
```

Comment:
Why arent we checking other fw versions like net core 2.2?

Reply:
This code uses the MSBuild runtime environment to guess whether _any_ version of `dotnet` is installed. Figuring out the version when the tool command of interest works with 2.1 or later installed is extra work for minimal benefit.